src/db/db_conn.cpp

```diff
@@ -1,22 +1,14 @@
 #include <string>
 #include <filesystem>
+
 #include "sqlite3.h"
 #include "db_conn.h"
 #include "common/log.h"
 
-#ifdef _MSC_VER
-#ifndef strcpy
-#define strcpy strcpy_s
-#endif
-#ifndef strncpy
-#define strncpy strncpy_s
-#endif
-#endif
-
 SQLite::SQLite(const char* path, const char* tag) 
 {
     sqlite3_open(path, &_db);
-    strcpy(this->tag, sizeof(this->tag), tag);
+    strncpy(this->tag, tag, sizeof(this->tag));
```

Comment:
This is regression, actually, since it doesn't null-terminate if size of of the `this->tag` buffer is exceeded.

Reply:
a8c665eab4485ac87c61ecc17140bafba739b6af
9ef4c7a7d52e46e0deeb1a3ed027e3b80dc5cbb6